### PR TITLE
Feat:header

### PR DESCRIPTION
**Pull request by TIEN1909**
#81 Responsive header

### CODE REVIEW

File: html-css-training/practice-two/src/styles/components/nav-item.css [hunk 1]
```diff
@@ -6,29 +6,48 @@
 }
 
 .header-nav-link {
+  box-sizing: border-box;
```
Comment (hungdo-agilityio) [inline html-css-training/practice-two/src/styles/components/nav-item.css:9]: Should this style belong to global?

File: html-css-training/practice-two/src/styles/layouts/header.css [hunk 2]
```diff
@@ -40,3 +40,43 @@
...
+    padding: 0 13px;
+  }
+  .header-nav-auth {
+    font-size: var(--fs-respon-x-small);
+    min-width: 110px;
+    height: 25px;
+  }
+}
+
+@media (min-width: 768px) and (max-width: 992px) {
```
Comment (hungdo-agilityio) [inline html-css-training/practice-two/src/styles/layouts/header.css:55]: 992px where come from?

File: html-css-training/practice-two/src/styles/components/nav-item.css [hunk 3]
```diff
@@ -9,26 +9,39 @@
...
-  display: none;
+.header-nav-link:hover {
+  border-top: 5px solid transparent;
+  border-bottom: 5px solid var(--secondary-bg-color);
 }
 
-.header-nav-link:hover::before {
-  display: flex;
-  justify-content: center;
+@media screen and (max-width: 992px) {
```
Comment (tungtran-agilityio) [inline html-css-training/practice-two/src/styles/components/nav-item.css:25]: any ideal create variable for breakpoint and reuse it?

Comment (TIEN1909) [reply html-css-training/practice-two/src/styles/components/nav-item.css:25]: you want me to create a variable for each <breakpoints>

Comment (tungtran-agilityio) [reply html-css-training/practice-two/src/styles/components/nav-item.css:25]: if you can, I think it does not support

**Review by tungtran-agilityio — APPROVED** [review]

**Pull request merged**